Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -134,7 +134,7 @@ public bool WriteToConsole (Size size, ExtendedCharInfo [] charInfoBuffer, Coord
         return result;
     }
 
-    public bool WriteANSI (string ansi)
+    internal bool WriteANSI (string ansi)
```

Comment:
This method sounds a lot like `internal bool WriteAnsiRequestDefault (string ansi)` whats the difference?

Reply:
I already removed the `WriteAnsiRequestDefault` method. They are different. The `WriteANSI` method call the `WindowsConsole.WriteConsole` of the kernel32.dll and the `WriteAnsiRequestDefault` was using the `Console.Out.Write`. On `WindowsDriver` we only can send requests through `WriteANSI` which is using the `WindowsConsole.WriteConsole`. If you use the `Console.Out.Write` we wont receive any response from the std-in, unless we are sending after the cleanup which restore the default console.

Reply:
> If you use the Console.Out.Write we wont receive any response from the std-in, unless we are sending after the cleanup which restore the default console.

I forgot to say that this is only happening in the `Windows Terminal Preview`.